cfme/fixtures/candu.py

```diff
@@ -0,0 +1,41 @@
+"""
+Fixtures for Capacity and Utilization
+"""
+import pytest
+from cfme.common.provider import BaseProvider
+from fixtures.provider import setup_or_skip
+
+
+@pytest.yield_fixture(scope="module")
+def enable_candu(appliance):
+    candu = appliance.collections.candus
+    server_settings = appliance.server.settings
+    original_roles = server_settings.server_roles_db
+    try:
+        server_settings.enable_server_roles(
+            'ems_metrics_coordinator',
+            'ems_metrics_collector',
+            'ems_metrics_processor'
+        )
+        server_settings.disable_server_roles(
+            'automate',
+            'smartstate'
+        )
+        candu.enable_all()
+
+        # ToDo: Its temporary wait for the metric collection.
+        # It should replace by wait_for with proper condition depends on the provider.
+        import time
+        time.sleep(900)
```

Comment:
**QUESTION**
What does prevent to implement a proper `wait_for`?

Reply:
@quarckster: For `recent hour` metric collection its took less time like some minutes but for `hourly` metric collection we have wait for some hours and for `daily` at least one day. Its not good idea to `sleep` our automation for a day. So its good to go with `DB dump` as per my understanding.

It's initial commit for C&U graph automation. I added temporary wait to merge `LineChart` Widget with simple test for `recent hour` data. After merging this block of code, I will discuss these problems with `Nandini` and we will move to next step for further enhancement.

@quarckster: I added changes as per your suggetion.